_posts/_2021-10-05-Launch-Announcement-1-1.markdown

```diff
@@ -0,0 +1,54 @@
+---
+layout: post
+authors: 
+  - henkle
+  - ahopp
+comments: true
+title: "OpenSearch 1.1 is here!"
+category:
+- releases
+---
+
+To the OpenSearch community everywhere, 
+
+We are excited to announce the 1.1 release of the OpenSearch project (available to [download](https://opensearch.org/downloads.html) today). Not only are we excited that this release marks the first minor version release on the OpenSearch project, it also comes with a number of new and exciting features. And, best of all, we continue to see the community grow and contribute to OpenSearch!
```

Comment:
use version permalink - in the future the download link will go to the wrong version.